extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json

```diff
@@ -0,0 +1,131 @@
+              "description": "AJO Campaign Description."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel": {
+          "title": "AJO Channel/Message Entity",
+          "type": "object",
+          "description": "AJO Channel/Message Entity Specific Fields. Denormalized on variantId",
+          "properties": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/email": {
+              "title": "Email Specific Fields",
+              "type": "object",
+              "description": "Email Specific Fields",
+              "properties": {
+                "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/email/subject": {
+                  "title": "Email Subject",
+                  "type": "string",
+                  "description": "Email subject, non-personalized"
+                }
+              }
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/push": {
+              "title": "Push Specific Fields",
+              "type": "object",
+              "description": "Push Specific Fields.",
+              "properties": {
+                "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/push/title": {
+                  "title": "Push Title",
+                  "type": "string",
+                  "description": "Push Title, non-personalized"
+                }
+              }
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/messageID": {
+              "title": "Message ID",
+              "type": "string",
+              "description": "Message ID. Does not change when message is edited."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/messageName": {
+              "title": "Message Name",
+              "type": "string",
+              "description": "Message Name. Refers to the referred version of messagePublciationID"
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/messagePublicationID": {
+              "title": "Message Publication ID",
+              "type": "string",
+              "description": "Message Publication ID. Represents a frozen/published version of message."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/variantID": {
+              "title": "Channel's variant ID",
+              "type": "string",
+              "description": "Is not frozen and might remain same between different versions of message publication."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/ajo/experiement": {
+          "title": "AJO Experiment Entity Specific Fields",
+          "type": "object",
+          "description": "AJO Experiment Entity Specific Fields",
+          "properties": {
```

Comment:
Don't we need experimentId here?
Is treatmentId an UUID?

Reply:
Please have this approved from one of your ajo SMEs once all your internal discussions resolved. Also, please leave a comment on this PR once its ready to be reviewed.